### PR TITLE
Remove HeliFlyCircle activity

### PR DESCRIPTION
**Pull request by reaperrr**
`FlyCircle` actually works fine for D2k Carryalls, and no other place used this.

### CODE REVIEW

File: OpenRA.Mods.Common/Traits/Air/Aircraft.cs [hunk 1]
```diff
@@ -621,9 +621,9 @@ protected virtual void OnBecomingIdle(Actor self)
 				self.QueueActivity(new TakeOff(self));
 			else if (Info.CanHover && self.Info.HasTraitInfo<AutoCarryallInfo>() && Info.IdleTurnSpeed > -1)
```
Comment (pchote) [inline OpenRA.Mods.Common/Traits/Air/Aircraft.cs]: Do we still need to check `AutoCarryallInfo` here? Isn't `Info.IdleTurnSpeed > -1` sufficient?

Comment (reaperrr) [reply OpenRA.Mods.Common/Traits/Air/Aircraft.cs]: It actually should be, yes.

**Comment by reaperrr** [comment]
Updated.

**Review by abcdefg30 — APPROVED** [review]

**Review by pchote — APPROVED** [review]

**Comment by pchote** [comment]
https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/07c919a09bb89236d9854ba69ea4ee3969457d29

**Pull request merged**